### PR TITLE
Set up most of the code for getting staff.

### PR DESCRIPTION
**Pull request by micah686**
I set up most of what was needed to run a "get staff" command. 
Using VndbFlags.Aliases currently breaks the program, so I have disabled the flag and model properties until someone else or I can figure out why it's causing a crash.

Other than that, the GetStaffAsync should be fully implemented.

### CODE REVIEW

File: VndbSharp/Models/Staff/Staff.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        public String Name { get; private set; }
+        [JsonProperty("original")]
+        public String OriginalName { get; private set; }
+        public Gender? Gender { get; private set; }
+		public String Language { get; private set; }
+        [JsonProperty("links")]
+		public StaffLinks StaffLinks { get; private set; }
+        public String Description { get; private set; }
+		//TODO: Need to fix staff aliases from breaking
+        public ReadOnlyCollection<StaffAliases> Aliases { get; private set; }
```
Comment (Nikey646) [inline VndbSharp/Models/Staff/Staff.cs]: Why exactly is this broken? I can't exactly accept this PR when something is broken. Whats the exception?

File: VndbSharp/Models/Staff/StaffAliases.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace VndbSharp.Models.Staff
+{
+    public class StaffAliases
+    {
+        internal StaffAliases(UInt32 Id, String Name, String OriginalName)
```
Comment (Nikey646) [inline VndbSharp/Models/Staff/StaffAliases.cs]: Where is this constructor used? The `ArrayOfArrayConverter` uses the constructor that only accepts an `JArray`

File: VndbSharp/Models/Staff/StaffLinks.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+using VndbSharp.Models.Common;
+
+namespace VndbSharp.Models.Staff
+{
+    public class StaffLinks: CommonLinks
+    {
+		[JsonProperty("homepage")]
```
Comment (Nikey646) [inline VndbSharp/Models/Staff/StaffLinks.cs]: Shouldn't be required.

File: VndbSharp/Models/Staff/StaffLinks.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+		public String Homepage { get; private set; }
+        [JsonProperty("twitter")]
```
Comment (Nikey646) [inline VndbSharp/Models/Staff/StaffLinks.cs]: Shouldn't be required.

File: VndbSharp/Models/Staff/StaffLinks.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        public String Twitter { get; private set; }
+        [JsonProperty("anidb")]
```
Comment (Nikey646) [inline VndbSharp/Models/Staff/StaffLinks.cs]: Shouldn't be required.

File: VndbSharp/Models/Staff/StaffLinks.cs [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        public String Anib { get; private set; }
```
Comment (Nikey646) [inline VndbSharp/Models/Staff/StaffLinks.cs]: Incorrectly spelt, should be AniDb

File: VndbSharp/Models/VndbFlags.cs [hunk 7]
```diff
@@ -63,7 +63,7 @@ public enum VndbFlags
 		///		<para>Valid on: Release, Character</para>
 		/// </summary>
 		[FlagIdentity("vn")]
-		VisualNovels = 1 << 7,
```
Comment (Nikey646) [inline VndbSharp/Models/VndbFlags.cs]: BC incompatible. Revert it.

File: VndbSharp/Models/VndbFlags.cs [hunk 8]
```diff
@@ -93,25 +93,52 @@ public enum VndbFlags
...
+	    /// </summary>
+		[FlagIdentity("voiced")]
+		Voiced = 1 << 12,
+
+	    /// <summary>
+	    ///		<para>Provides a list of VNs this staff has worked on</para>
+	    ///		<para>Valid on: Staff</para>
+	    /// </summary>
+	    [FlagIdentity("vns")]
+	    VisualNovels = 1 << 13,
```
Comment (Nikey646) [inline VndbSharp/Models/VndbFlags.cs]: Use the work around used for characters, having two "VisualNovel(s)" flags is ambiguous, and has purposely been avoided.

File: VndbSharp/Models/VndbFlags.cs [hunk 9]
```diff
@@ -93,25 +93,52 @@ public enum VndbFlags
...
+
+		/// <summary>
+		///		<para>Provides the aliases of the staff who worked on the VN</para>
+		///		<para>Valid on: Staff</para>
+		/// </summary>
+		//[FlagIdentity("aliases")]
+		//Aliases = 1 << 14,
```
Comment (Nikey646) [inline VndbSharp/Models/VndbFlags.cs]: Refer to comment in `VndbSharp/Models/Staff/Staff.cs`

**Comment by micah686** [comment]
Ok, see if those changes that I made are acceptable.

File: VndbSharp/Models/VndbFlags.cs [hunk 10]
```diff
@@ -63,7 +63,7 @@ public enum VndbFlags
 		///		<para>Valid on: Release, Character</para>
 		/// </summary>
 		[FlagIdentity("vn")]
-		VisualNovel = 1 << 7,
+		VisualNovels = 1 << 7,
```
Comment (Nikey646) [inline VndbSharp/Models/VndbFlags.cs]: Missing the work around used with characters, in [VndbUtils](https://github.com/Nikey646/VndbSharp/blob/master/VndbSharp/VndbUtils.cs#L133)

Comment (micah686) [inline VndbSharp/Models/VndbFlags.cs]: Did that last commit fix the issue?

**Pull request merged**